src/qibolab/_core/instruments/qblox/config.py

```diff
@@ -59,10 +60,13 @@ def local_address(self):
             https://docs.qblox.com/en/main/api_reference/cluster.html#qblox_instruments.Cluster.connect_sequencer
         """
         direction = "in" if self.input else "out"
+        for port in self.ports:
+            if port is not None:
+                assert port > 0
         channels = (
-            str(self.ports[0])
+            str(self.ports[0] - 1)
```

Comment:
Otherwise I get
```py
ValueError: connection command 'out4' (index 0): dac is out of range: 4
```
and after playing a bit I realized that available `out`s go from 0 to 3.